neqo-transport/src/fc.rs

```diff
@@ -286,36 +300,46 @@ impl ReceiverFlowControl<()> {
         tokens: &mut Vec<RecoveryToken>,
         stats: &mut FrameStats,
     ) {
-        if let Some(max_allowed) = self.frame_needed() {
-            if builder.write_varint_frame(&[FRAME_TYPE_MAX_DATA, max_allowed]) {
-                stats.max_data += 1;
-                tokens.push(RecoveryToken::MaxData(max_allowed));
-                self.frame_sent(max_allowed);
-            }
+        if !self.frame_needed() {
+            return;
+        }
+        if builder.write_varint_frame(&[FRAME_TYPE_MAX_DATA, self.next_limit()]) {
+            stats.max_data += 1;
+            tokens.push(RecoveryToken::MaxData(self.next_limit()));
+            self.frame_sent(self.next_limit());
         }
     }
 }
 
 impl ReceiverFlowControl<StreamId> {
+    /// Double the active limit every time we need to send an update.
+    fn increase_active(&mut self) {
+        if self.active < self.max_active {
+            self.active = min(self.max_active, self.active * 2);
```

Comment:
I think we need to have a higher initial limit, otherwise we will have a slow download for a couple of RTTs and probably get into app-limited cc state.
My initial idea was to have a low initial value in the transport parameter, but to increase it immediately to something high for the streams from the focus tab and/or high priority items. This kind of change will need investigation.

I would suggest just to bump the initial value to something high.(maybe 12MB that is HTTP2 limit; HTTP2 also was the same session limit). This is simpler and I do not see a need for more complex solution.